packages/protocol/thread/thread.ts

```diff
@@ -266,19 +264,8 @@ class _ThreadFront {
     await this.waitForSession();
     await this.initializedWaiter.promise;
     await this.ensureAllSources();
-    this.ensureCurrentPause();
-
-    if (this.testName) {
-      await gToolbox.selectTool("debugger");
-      window.Test = await import("test/harness");
-      const script = document.createElement("script");
-      script.src = `/test/scripts/${this.testName}`;
-      document.head.appendChild(script);
-    }
-  }
```

Comment:
Replay devtools _e2e test code_ definitely didn't belong in the protocol package.